lib/Command/SetSecretVerificationStatusCommand.php

```diff
@@ -0,0 +1,150 @@
+<?php
+/**
+ * @author Semih Serhat Karakaya <karakayasemi@itu.edu.tr>
+ *
+ * Two-factor TOTP
+ *
+ * This code is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License, version 3,
+ * as published by the Free Software Foundation.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License, version 3,
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>
+ *
+ */
+
+namespace OCA\TwoFactor_Totp\Command;
+
+use InvalidArgumentException;
+use OCA\TwoFactor_Totp\Db\TotpSecretMapper;
+use OCP\AppFramework\Db\DoesNotExistException;
+use OCP\IUserManager;
+use Symfony\Component\Console\Command\Command;
+use Symfony\Component\Console\Input\InputArgument;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Input\InputOption;
+use Symfony\Component\Console\Output\OutputInterface;
+
+class SetSecretVerificationStatusCommand extends Command {
+
+	/** @var TotpSecretMapper */
+	private $secretMapper;
+
+	/** @var IUserManager */
+	private $userManager;
+
+	public function __construct(
+		TotpSecretMapper $secretMapper,
+		IUserManager $userManager
+	) {
+		parent::__construct();
+		$this->secretMapper = $secretMapper;
+		$this->userManager = $userManager;
+	}
+
+	protected function configure() {
+		$this
+			->setName('twofactor_totp:set-secret-verification-status')
+			->setDescription(
+				'Set secret verification of specified users or all users'
+			)
+			->addArgument(
+				'set-verified',
+				InputArgument::REQUIRED,
+				'Secret verification status to set. (true or false)'
+			)
+			->addOption(
+				'all',
+				null,
+				InputOption::VALUE_NONE,
+				'Will affects all users secrets'
+			)
+			->addOption(
+				'uid',
```

Comment:
In many other ``occ`` commands we also have a group option, so that things can be managed for groups of users.
Is that relevant/useful to add here?
(It should be quite easy to add a group option like elsewhere)

Reply:
The command is written to fix this issue: #46. IMHO, it has not much more use-case other than this fix. So, we can we can leave it as it is.
Just maybe, admins can use this command to force members of a group to refresh their secret. For this case, if you want I can add "group" option. @PVince81 any thoughts?